packages/indexer-client/src/IndexerBaseClient.ts

```diff
@@ -814,6 +816,20 @@ export class IndexerBaseClient {
     };
   }
 
+  /**
+   * Retrieves VRTX total / circulating supply
+   *
+   * @param params
+   */
+  async getVrtxTokenInfo(
+    params: GetIndexerVrtxTokenInfoParams,
+  ): Promise<GetIndexerVrtxTokenInfoResponse> {
+    const baseUrl = this.opts.url.replace('/v1', '');
```

Comment:
Should we have some sort of `this.v2Url` for when we add more queries in the future?

We should address types as well but can do that later.

Would we be able to just replace /v1 with /v2?

Reply:
yea, will do!

Reply:
addressed!